components/analytics/compact-dashboard.tsx

```
@@ -0,0 +1,383 @@
+                />
+
+                <div className="flex gap-2 ml-2">
+                    {/* Quick time range filters */}
+                    <div className="hidden md:flex items-center gap-1">
+                        {timeRanges.map((range) => (
+                            <Button
+                                key={range.value}
+                                variant="outline"
+                                size="sm"
+                                className={`text-xs ${dateRange?.startDate && dateRange.endDate &&
+                                    dayjs(dateRange.endDate).diff(dayjs(dateRange.startDate), 'day') === parseInt(range.value.replace('d', '')) - 1
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)